src/style/search/search-bar.less

```diff
@@ -44,6 +43,19 @@
     overflow: visible;
 }
 
+.main__no-panel .search-bar__dialog {
+    margin-left: -@search-bar-width / 2;
+}
+
+
+.main__one-panel .search-bar__dialog {
+    margin-left: -(@properties-width / 2)  - (@search-bar-width / 2);
```

Comment:
No, it is a CSS variable (it is the width of one panel...maybe we should rename that)